clang_delta/ExpressionDetector.cpp

```diff
@@ -64,7 +64,7 @@ class IncludesPPCallbacks : public PPCallbacks {
                           StringRef FileName, bool IsAngled,
                           CharSourceRange FilenameRange, const FileEntry *File,
                           StringRef SearchPath, StringRef RelativePath,
-                          const Module *Imported) override;
+                          const Module *Imported);
```

Comment:
This doesn't look correct. We should follow the same API as in LLVM 7.0, e.g.,

    https://github.com/csmith-project/creduce/commit/c0d9c3a2ef763eb09f0a70b8ad86250598549487

Reply:
Well, if you believe clang is wrong to give that error in this case then we need to disable that error for clang in the build scripts.

Reply:
Oh I see, yes.